File: Angular/src/app/models/Lead.ts
import { LeadSource } from '../models/LeadSource'
import { LeadStatus } from '../models/LeadStatus'

export class Lead {

	id!: String;
    firstName!: any;
    lastName!: any;
    email!: any;
    phone!: any;
    picture!: any;
    leadSource!: LeadSource ;
    status !: LeadStatus;


}

File: crmmicroservice/src/main/java/tn/esprit/spring/models/Lead.java
package tn.esprit.spring.models;

import javax.persistence.Id;


import java.io.Serializable;
import java.util.Date;
import java.util.List;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;


@Data
@Entity
@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
public class Lead implements Serializable{
	
	@Id
	@GeneratedValue
	private Long id ;
	


    private String firstName ;

    private String lastName ;

    private String email ;

    private Number phone ;

    

    private LeadStatus status;

    private LeadSource leadSource;
    


}
